### ISSUE TITLE
DynatraceLookupPlugin logs error on startup when magic metadata file isn't available

### ISSUE THREAD

**Issue by checketts** [michaljonko/dynatrace-log4j2-appender#3]
When starting my application for local development (in my IDE, which doesn't have a one agent available), the Dynatrace Log4j appender logs the following:

```
ERROR StatusLogger DynatraceLookup cannot read metadata (magic file dt_metadata_e617c525669e072eebe3d0f08212e8f2.properties)
```

I would like a way to disable that when doing local development, even though it doesn't affect my app.

Is this repo accepting pull requests?

**Comment by dbelyaev** [comment]
@checketts
Looking for the same solution.
You are talking about PR, do you have a working example to disable metadata lookup?

**Comment by checketts** [comment]
@dbelyaev  No, I decided to use the OpenTelemetry logback appender. I had been using Logback before anyways, so that reduced the changes my projects required.
